pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go

```diff
@@ -0,0 +1,82 @@
+package k8s_runtime
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const appConf = `
```

Comment:
add t or test or testxxx as prefix is better for global testing variables

Reply:
ok